pkg/cmd/root.go

```diff
@@ -153,18 +152,12 @@ func ensureConfigDefaults() {
 	}
 
 	if needsWrite {
```

Comment:
is needsWrite always true? It seems to always run

Reply:
I know mine definitely doesn't..

Reply:
Running nitric version on a build from develop
![image](https://user-images.githubusercontent.com/13415171/156265325-8289c457-e0f3-45e1-aa81-34f09e3c018e.png)

Reply:
Running version on the latest dev build:
![image](https://user-images.githubusercontent.com/13415171/156265480-ba15098c-6b59-4459-92e8-ea1466d05b58.png)

I know removing the tasklet will remove the output, but it does run every time on my machine.